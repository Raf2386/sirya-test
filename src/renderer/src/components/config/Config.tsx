import { ReactElement } from 'react';

const Config = (): ReactElement => {
  return (
    <div id="config">
      <h1>CIAO RAFFA !</h1>
    </div>
  );
}

export default Config;
