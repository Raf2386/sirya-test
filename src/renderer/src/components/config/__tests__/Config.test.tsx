import '@testing-library/jest-dom'
import { render } from "@testing-library/react"

import Config from "../Config"

describe('Config', () => {
  describe('Rendering', () => {
    test('Renders the config page', () => {
      const { asFragment } = render(<Config />)
      expect(asFragment()).toMatchSnapshot()
      expect(asFragment().firstChild).toHaveTextContent('CIAO RAFFA !')
    })
  })
  describe('Functionality', () => {
    test('TEST JEST', () => {
      expect(true).toBe(true)
    })
  })
})
