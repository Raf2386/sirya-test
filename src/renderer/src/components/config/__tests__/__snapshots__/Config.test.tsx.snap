// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config Rendering Renders the config page 1`] = `
<DocumentFragment>
  <div
    id="config"
  >
    <h1>
      CIAO RAFFA !
    </h1>
  </div>
</DocumentFragment>
`;
